### PR TITLE
More fixes

### PR DESCRIPTION
**Pull request by djpnewton**
- hide permissions stuff email login (we arent using it)
 - fix layout of dashboard links in admin dashboard (the alignment is off)
 - fix layout of lightning dashboard links
 - fix rounding of dashboard balances (https://be.bitforge.me/reporting/dashboard_general)
 - fix error showing order history on prod because of unsupported assets (DOGE)

### CODE REVIEW

**Review by eoliveros — COMMENTED** [review]
the following is what ive observed:
- login page appears to have shifted to the left of the screen. previously it was centered.
- admin index appears to be have shifted on the left of the screen as well.
- appears all the forms associated with the buttons on appears to have shifted to left of screen with the exception of flask-admin home.
- dashboard_general shows 2 decimal points for NZD and 8 decimal points for BTC
- lightning home has been shifted to the left as well and is centered within the given frame?

if the above is correct, i can approve it.

**Comment by djpnewton** [comment]
you probably want to review the 4 individual commits, they each address an issue listed in the PR description

the HTML/CSS changes are mainly about simplifying the HTML and CSS and using more of the built in bootstrap stuff

**Review by eoliveros — COMMENTED** [review]
there's no link between the dashboard_report_broker_order and flask-admin.

File: src/templates/reporting/dashboard_broker_orders.html [hunk 1]
```diff
@@ -1,108 +1,67 @@
...
         </div>
-        {% for market in orders_data %}
-            <div class="row">
-                <div class="column">
-                    <div class="row-2line-title white text_color report-row-padding-1line">{{ orders_data[market].asset_symbol }}</div>
-                    <div class="hide">No. of orders ( amount in {{ orders_data[market].asset_symbol }} )</div>
-                </div>
-                <div class="column">
-                    <a href="/admin/brokerorder/?flt1_0={{ orders_data[market].today }}+to+{{ orders_data[market].tomorrow }}&flt2_1={{ orders_data[market].asset_symbol }}">
-                        <div class="report-row report-row-hover white text_color report-row-padding-2line">
```
[the remark this answers is not in the document]
Comment (djpnewton) [reply src/templates/reporting/dashboard_broker_orders.html:53]: re-added the missing links

**Review by eoliveros — APPROVED** [review]
Looks good

**Pull request merged**